lib/utils.js

```diff
@@ -1,6 +1,19 @@
 // Custom Splitter:
-const split = (str, delimiter) => {
-    return (str.split(delimiter || ',').map(element => element.trim())).filter(element => element);
+const split = (str, delimiter, whitespace) => {
```

Comment:
Can we name `whitespace` better? Something that makes it clear that it's a boolean. Like `with_plus_for_whitespace`.

Reply:
replaced it with `haswhitespace`